common/config/data.tsx

```diff
@@ -48,7 +48,7 @@ export const gasPriceDefaults: GasPriceSetting = {
 
 export const MINIMUM_PASSWORD_LENGTH = 12;
 
-export const knowledgeBaseURL = 'https://support.mycrypto.com';
+export const knowledgeBaseURL = 'https://support.mycrypto.com/';
```

Comment:
Double check this one, you're going to end up with a _lot_ of double slashes in a lot of places, which will be an invalid URL. I'd advise leaving it as-is.